libbpfgo.c

```diff
@@ -189,15 +191,10 @@ void cgo_bpf_object_open_opts_free(struct bpf_object_open_opts *opts)
     free(opts);
 }
 
-struct bpf_map_create_opts *cgo_bpf_map_create_opts_new(__u32 btf_fd,
-                                                        __u32 btf_key_type_id,
-                                                        __u32 btf_value_type_id,
-                                                        __u32 btf_vmlinux_value_type_id,
-                                                        __u32 inner_map_fd,
-                                                        __u32 map_flags,
-                                                        __u64 map_extra,
-                                                        __u32 numa_node,
-                                                        __u32 map_ifindex)
+struct bpf_map_create_opts *cgo_bpf_map_create_opts_new(
+    __u32 btf_fd, __u32 btf_key_type_id, __u32 btf_value_type_id,
+    __u32 btf_vmlinux_value_type_id, __u32 inner_map_fd, __u32 map_flags,
+    __u64 map_extra, __u32 numa_node, __u32 map_ifindex)
```

Comment:
That's other oddity (ugliness)... some functions will align its params using the first line, others like this will break in the next.